hudi-sync/hudi-hive-sync/src/main/java/org/apache/hudi/hive/HiveSyncTool.java

```diff
@@ -169,25 +168,24 @@ protected void syncHoodieTable(String tableName, boolean useRealtimeInputFormat,
     boolean isDropPartition = hoodieHiveClient.isDropPartition();
 
     // check if schemaChanged
-    boolean schemaChanged = false;
-
-    if (!isDropPartition) {
```

Comment:
This judgment is the reason that the manual execution of HiveSyncTool does not update the schema.